utils/io/logger.go

```diff
@@ -0,0 +1,120 @@
+package io
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"io/ioutil"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+
+	cioutil "github.com/kubernetes-sigs/cri-o/utils/ioutil"
+)
+
+const (
+	// delimiter used in CRI logging format.
+	delimiter = ' '
+	// eof is end-of-line.
```

Comment:
super duper small and largely inconsequential nit: did you mean eof or eol here?

Reply:
Code duplicated from containerd/cri go packages. I think you're right regarding the comment though!